diff --git a/relay/kusama/Cargo.toml b/relay/kusama/Cargo.toml
index 4b5dd7e7f..ab10fd408 100644
--- a/relay/kusama/Cargo.toml
+++ b/relay/kusama/Cargo.toml
@@ -106,6 +106,7 @@ xcm-runtime-apis = { workspace = true }
 
 sp-debug-derive = { workspace = true }
 relay-common = { workspace = true }
+pallet-sudo = { workspace = true }
 
 [dev-dependencies]
 sp-keyring = { workspace = true }
@@ -178,6 +179,7 @@ std = [
 	"pallet-society/std",
 	"pallet-staking-runtime-api/std",
 	"pallet-staking/std",
+	"pallet-sudo/std",
 	"pallet-timestamp/std",
 	"pallet-transaction-payment-rpc-runtime-api/std",
 	"pallet-transaction-payment/std",
diff --git a/relay/kusama/src/genesis_config_presets.rs b/relay/kusama/src/genesis_config_presets.rs
index 889575460..0c52f79fd 100644
--- a/relay/kusama/src/genesis_config_presets.rs
+++ b/relay/kusama/src/genesis_config_presets.rs
@@ -193,6 +193,9 @@ fn kusama_testnet_genesis(
 			"forceEra": Forcing::NotForcing,
 			"slashRewardFraction": Perbill::from_percent(10),
 		},
+		"sudo": {
+			"key": Some(_root_key),
+		},
 		"babe": {
 			"epochConfig": Some(BABE_GENESIS_EPOCH_CONFIG),
 		},
diff --git a/relay/kusama/src/lib.rs b/relay/kusama/src/lib.rs
index d158600ad..8e7126b14 100644
--- a/relay/kusama/src/lib.rs
+++ b/relay/kusama/src/lib.rs
@@ -1892,9 +1892,17 @@ construct_runtime! {
 		// refer to block<N>. See issue #160 for details.
 		Mmr: pallet_mmr = 201,
 		BeefyMmrLeaf: pallet_beefy_mmr = 202,
+
+		// Sudo.
+		Sudo: pallet_sudo = 255,
 	}
 }
 
+impl pallet_sudo::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
+}
 /// The address format for describing accounts.
 pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
 /// Block header type as expected by this runtime.
diff --git a/relay/polkadot/Cargo.toml b/relay/polkadot/Cargo.toml
index 9b3d1dc46..5fed90096 100644
--- a/relay/polkadot/Cargo.toml
+++ b/relay/polkadot/Cargo.toml
@@ -108,6 +108,8 @@ sp-debug-derive = { workspace = true }
 # just for the coretime migration
 polkadot-parachain-primitives = { workspace = true }
 
+pallet-sudo = { workspace = true }
+
 [dev-dependencies]
 approx = { workspace = true }
 sp-keyring = { workspace = true }
@@ -178,6 +180,7 @@ std = [
 	"pallet-staking-runtime-api/std",
 	"pallet-staking/std",
 	"pallet-state-trie-migration/std",
+	"pallet-sudo/std",
 	"pallet-timestamp/std",
 	"pallet-transaction-payment-rpc-runtime-api/std",
 	"pallet-transaction-payment/std",
diff --git a/relay/polkadot/src/genesis_config_presets.rs b/relay/polkadot/src/genesis_config_presets.rs
index b8f828d7a..77487f0eb 100644
--- a/relay/polkadot/src/genesis_config_presets.rs
+++ b/relay/polkadot/src/genesis_config_presets.rs
@@ -193,6 +193,9 @@ fn polkadot_testnet_genesis(
 			"forceEra": Forcing::NotForcing,
 			"slashRewardFraction": Perbill::from_percent(10),
 		},
+		"sudo": {
+			"key": Some(_root_key),
+		},
 		"babe": {
 			"epochConfig": Some(BABE_GENESIS_EPOCH_CONFIG),
 		},
diff --git a/relay/polkadot/src/lib.rs b/relay/polkadot/src/lib.rs
index 3b2275e68..a26ec7157 100644
--- a/relay/polkadot/src/lib.rs
+++ b/relay/polkadot/src/lib.rs
@@ -1651,9 +1651,18 @@ construct_runtime! {
 		// refer to block<N>. See issue #160 for details.
 		Mmr: pallet_mmr = 201,
 		BeefyMmrLeaf: pallet_beefy_mmr = 202,
+
+		// Sudo.
+		Sudo: pallet_sudo = 255,
 	}
 }
 
+impl pallet_sudo::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type WeightInfo = ();
+}
+
 /// The address format for describing accounts.
 pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
 /// Block header type as expected by this runtime.
